docusign/authmetamodel.go

```diff
@@ -0,0 +1,28 @@
+package docusign
+
+import (
+	"github.com/amp-labs/connectors/common/paramsbuilder"
+)
+
+type CatalogVariables struct {
```

Comment:
This struct makes the structure returned by PostAuthInfo formal. It is used bidirectionaly when getting, setting metadata.

Since we want ProviderInfo to fill in `{{.server}}` placeholder this struct implements substitution.